app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -317,23 +317,23 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
         className={['usa-label', shownErrorMessage && 'usa-label--error'].filter(Boolean).join(' ')}
       >
         {label}
+        <StatusMessage status={Status.ERROR}>{shownErrorMessage}</StatusMessage>
+        <StatusMessage
```

Comment:
Placing these in the `label` makes it the _name_ of the field. I don't think an error message is appropriate as part of the name of the field.

Elsewhere, we'll usually add a `aria-describedby` association when a field has associated error text. Could we consider that here?

Reply:
That makes much more sense. I'll revert these changes in this PR and add the `aria-describedby` as suggested. Now that I'm looking at `aria-describedby` I'm seeing that we do already use it in this file, but it might need changes so `aria-describedby` can sometimes point to the `hint` and sometimes the `StatusMessage` (field error). I'm investigating.

Reply:
👍 It's valid to specify multiple (space-separated) IDs as part of `aria-describedby`. They should be read in order by the screen reader after announcing the name of the field, with a brief pause between.

Reply:
That did make things very doable. Latest version of this work changes the aria-describedby for the file input to include the `StatusMessage`s for error and success display.